Chat/WebApi/Program.cs

```diff
@@ -0,0 +1,84 @@
+using Application.Dtos;
+using Application.Extentions;
+using Application.Ports.Services;
+using Application.Services;
+using Application.Validators;
+using Domain.Interfaces;
+using FluentValidation;
+using Infrastructure.Data;
+using Infrastructure.Repositories;
+using Microsoft.EntityFrameworkCore;
+using FluentValidation.AspNetCore;
+using Application.AutoMapperProfiles;
+
+namespace WebApi
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+            builder.Services.AddDbContext<DatabaseContext>(options =>
+            {
+
+                options.UseSqlServer(
+                    builder.Configuration.GetConnectionString("Default"),
+                    builder => builder.MigrationsAssembly("Infrastructure"));
+                options.LogTo(Console.WriteLine);
+            });
+
+            builder.Services.AddAutoMapper(typeof(ChatsProfile).Assembly);
+            builder.Services.AddAutoMapper(typeof(MessageProfile).Assembly);
+            
```

Comment:
There's no need to write it twice. For one mapper you write, the rest will automatically be included